### PR TITLE
docs: poetry, env variables, favicon

### PR DESCRIPTION
**Pull request by siyangqiu**
## 📥 Pull Request

**📘 Description**
A variety of doc improvements

### CODE REVIEW

File: docs/v1/details/environmental-variables.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: "Environmental Variables"
```
Comment (HowieG) [inline docs/v1/details/environmental-variables.mdx]: rename to environment variables

File: docs/v1/details/tracking-llm-calls.mdx [hunk 2]
```diff
@@ -3,8 +3,21 @@ title: "Tracking LLM Calls"
 description: "Tracking LLM Calls using the AgentOps SDK"
 ---
 
-When the AgentOps SDK detects the `openai` as an installable module, it will automatically
-start tracking OpenAI calls by instrumenting chat completions to track their usage.
+### How it works
+
+When the AgentOps SDK detects the `openai`, `litellm`, or `cohere` as installed modules, it will automatically
```
Comment (HowieG) [inline docs/v1/details/tracking-llm-calls.mdx:8]: I think we should maintain a list somewhere of models we support first-class and then do when you import openai or <link>any of the models we support first-class</link>. If we maintain this list separately we'll forget to update it and it'll cause confusion

File: docs/v1/details/tracking-llm-calls.mdx [hunk 3]
```diff
@@ -3,8 +3,21 @@ title: "Tracking LLM Calls"
...
+start tracking their usage. No further work is required from you! 😊
+
+### Not working? 
+
+Try these steps:
+1. Make sure you have the latest version of the AgentOps SDK installed. We are constantly updating it to support new LLM libraries and releases.
+2. Make sure you are calling `agentops.init()` *after* importing the LLM module but *before* you are calling the LLM method.
```
Comment (HowieG) [inline docs/v1/details/tracking-llm-calls.mdx:15]: nice

File: docs/v1/details/tracking-llm-calls.mdx [hunk 4]
```diff
@@ -3,8 +3,21 @@ title: "Tracking LLM Calls"
...
+3. Make sure the `instrument_llm_calls` parameter of `agentops.init()` is set to `True`.
```
Comment (HowieG) [inline docs/v1/details/tracking-llm-calls.mdx]: is set to "True" (default)

File: docs/v1/details/tracking-llm-calls.mdx [hunk 5]
```diff
@@ -3,8 +3,21 @@ title: "Tracking LLM Calls"
...
+
+Still not working? Please let us know! You can find us on [Discord](https://discord.gg/DR2abmETjZ),
+ [GitHub](https://github.com/AgentOps-AI/agentops), 
+ or email us at [shawn@agentops.ai](mailto:shawn@agentops.ai).
```
Comment (HowieG) [inline docs/v1/details/tracking-llm-calls.mdx]: should we have a eng@agentops.ai ?

**Review by HowieG — CHANGES_REQUESTED** [review]

File: docs/v1/details/tracking-llm-calls.mdx (hunk 2 above)
Comment (siyangqiu) [reply docs/v1/details/tracking-llm-calls.mdx:8]: Let's just push this for now. We can do that in a later change

File: docs/v1/details/tracking-llm-calls.mdx (hunk 5 above)
Comment (siyangqiu) [reply docs/v1/details/tracking-llm-calls.mdx]: I can make one, but it will just go to my email

**Review by HowieG — APPROVED** [review]

**Pull request merged**